### PR TITLE
at.cpp, pc.cpp, genpc.cpp: adding BIOS versions

### PR DESCRIPTION
**Pull request by rfka01**
Added BIOS revisions to pc and pcv20, NCR PC6 [eggimac], AUVA VIP 800 [fdiskitup], Hyundai Super 16T and TE [sequoia], Octec XT-286 [ozzyrules] and Nixdorf 8810-20 [MajorMaxdom]

### CODE REVIEW

File: src/mame/pc/pc.cpp [hunk 1]
```diff
@@ -2338,6 +2384,63 @@ ROM_START( mpu9088vf ) // From a motherboard marked MY-COM MPU-9088-VF SAN-MS94V
...
+OSC: 24.000 MHz, 14.31818
+ISA: 8 slots
+
+*****************************************************************************/
+
+ROM_START( auvip800 ) // a v2.30 Juko BIOS exists on this MB, cf. juko8
+	ROM_REGION(0x10000, "bios", 0) 
+	ROM_LOAD( "bxm10_phoenix_ver 2.52d.bin", 0xe000, 0x2000, CRC(9c964c80) SHA1(59a60425aa867abd33d30303300ed3c587969d2a))
+ROM_END
 } // anonymous namespace
```
Comment (cuavas) [inline src/mame/pc/pc.cpp:2444]: Shouldn’t there be a blank line there?

**Comment by rfka01** [comment]
Thanks for reviewing. The line hat gotten eaten by manually moving things to the 0.256 codebase.

**Comment by cuavas** [comment]
Where do these come from?  New machines should be credited.

**Comment by rfka01** [comment]
The submitted files have all the credits, just not the commit message.

**Comment by Tafoid** [comment]
@rfka01 The provided data romset for machine **ocxt286** does not appear to be complete.

In particular, missing one of the two added to source BIOS entries.  The missing one is:

`ROM_SYSTEM_BIOS(1,` "ocxt286_2", "Hedaka HED-919")
`ROMX_LOAD(` "hed919-328f.bin", 0x1e000, 0x2000, CRC(15117381) SHA1(d9dfa796edf7e94b9dacf984763ac046cf80f26d), `ROM_BIOS(1))`

Please upload as normal to fill ASAP.

Thank you.

**Pull request merged**